src/assets/css/sections/hero_section.css

```diff
@@ -0,0 +1,34 @@
+.hero_section {
+  display: flex;
+  flex-direction: row;
+  margin-top: 25px;
+  margin-left: 145px;
+  margin-right: 150px;
+}
```

Comment:
Between block codes, you should have a new line

Reply:
Thank you, I fixed it